src/constructs/core/custom-resources/runtime/lambda.test.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * You can't specify the file extension (eg .ts) of the file to import,
+ * and lambda.js exists as a dummy file, so we'd always import that over lambda.ts (the actual code to be tested) if
+ * we imported `lambda`
+ * lambda.symlink is a symlink to lambda.ts, so we can test it without importing lambda.js
+ * */
+import { flatten } from "./lambda.symlink";
```

Comment:
Oh wow! I'd be curious to know the output of `npm pack` now? IIUC [`npm pack`](https://docs.npmjs.com/cli/v6/commands/npm-pack) will create a local bundle that's representative of what get's published to NPM - a sort of publish dry run.

I'm not sure what JS file will be added - the empty lambda.js or the one that tsc will create?

Reply:
Just ran it and it looks like it packs fine! By that I mean the resulting `lambda.js` file is the expected compiled TS lambda, not the dummy file. I assume it simply overwrites it no matter what, as the dummy file could equally be seen as a stale JS file in need of updating